sql/sql_parse.cc

```diff
@@ -1437,7 +1437,7 @@ bool dispatch_command(THD *thd, const COM_DATA *com_data,
       else
       {
         mysql_reset_thd_for_next_command(thd);
-        my_error(ER_LOCK_DEADLOCK, MYF(0), "wsrep aborted transaction");
+        my_message(ER_LOCK_DEADLOCK, "WSREP cancelled the transaction", MYF(0));
```

Comment:
I am wondering if we should be introducing a new error code to capture WSREP operation failure.
ER_WSREP_XXXX.

This will help easy classification of error that are raised by WSREP module.